openadapt/record.py

```diff
@@ -37,6 +39,43 @@
 
 Event = namedtuple("Event", ("timestamp", "type", "data"))
 
+utils.configure_logging(logger, LOG_LEVEL)
+
+
+def args_to_str(*args):
+    return ", ".join(map(str, args))
+
+
+def kwargs_to_str(**kwargs):
+    ret = ""
+    for k, v in kwargs.items():
+        ret += f"{k}={v},"
+
+    return ret[:-1]
+
+
+def logging(logger):
```

Comment:
Maybe we should name this differently, e.g. `trace(logger)`, what do you think?

Reply:
renamed in [db24c49](https://github.com/MLDSAI/OpenAdapt/pull/229/commits/db24c49d6d83631b825b6706c3efa316f9ac5030)